packages/@lwc/engine-server/src/__tests__/fixtures/component/modules/x/component/component.html

```diff
@@ -1,3 +1,4 @@
 <template>
     <x-child></x-child>
+    <x-anotherchild></x-anotherchild>
```

Comment:
I don't see a test here for `<lwc:component>` ...?

BTW this is a good example of why it'd be nice to have the compiler output as part of the snapshots... would help with visualizing the generated code.

Reply:
This tests the import statement we add when we see a child in the template. `lwc:component` are imported in the JS by the user.

Specifically, here https://github.com/salesforce/lwc/pull/4000/commits/db3322fa2aa8af2f0b5ade1f24b169b09142b65f#diff-17a5fc2e6a05c3f1e674d6942273c831cb914d8d3c3286062c429e5054465bdfR101, we increment UID to add unique import statements for each child.
e.g.
```js
import { generateMarkup as generateMarkup_1 } from 'x/child';
import { generateMarkup as generateMarkup_2 } from 'x/anotherchild';
```

Definitely agree with having the compiler output in snapshots. This would have been clearer.